codegen/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "gluon_codegen"
+version = "0.7.1" # GLUON
+authors = ["Markus <marwes91@gmail.com>"]
+
+license = "MIT"
+
+description = "Code generation macros for the gluon programming language"
+
+repository = "https://github.com/gluon-lang/gluon"
+documentation = "https://docs.rs/gluon"
+
+[lib]
+proc-macro = true
+
+[dependencies]
+syn = "0.14"
+quote = "0.6"
+proc-macro2 = "0.4"
+
+gluon = { version = "0.7.1", path = ".." } # GLUON
```

Comment:
I don't think gluon is actually required for this crate, right? (Only the generated code refer to it)

Having the dev-dependencis on gluon is ok though.

Reply:
I just checked, I do actually use gluon for the `VmTag` typedef. I can just replace it with `u32` of course, but that would kind defeat the purpose.

Reply:
I think it is fair to just use `usize` for it. If someone happens to have more than `VmTag` variants then they will still get a warning (if they somehow manage to compile a 4>GB file that is :). Not needing the dependency is nice though as it opens up the possibility to use this crate in the gluon and gluon_vm crates aswell.

Reply:
Yeah, fair enough, I'll remove it :D